src/000-SCRIPT_OBJ/Tasks.js

```diff
@@ -143,7 +143,7 @@ App.Task = class Task {
                     }
                     break;
                 case "NPC_STAT":
-                    if (typeof NPC === 'undefined' && typeof Option !== 'undefined') NPC = Player.GetNPC(Option);
+                    if (typeof Option !== 'undefined') NPC = Player.GetNPC(Option);
```

Comment:
@JJM152, please check this. No job "CHECK" in the game so far overwrites NPC name using "OPT".

Reply:
It's not ringing a bell for me. I'm just going to push this through and continue testing while I work on finishing up this dice game.

Reply:
OK, I continue too.